src/packagedcode/pypi.py

```diff
@@ -793,6 +794,11 @@ def parse(cls, location):
                 )
             ]
 
+        extracted_license_statement = metadata.get('license')
+        license_file_references = metadata.get('license_files')
```

Comment:
Do we need to "follow" these license file references? of this does not matter since they would end up in the wheel .dist-info anyway?

Reply:
We are following this for now. Added new issue to track this separately : https://github.com/nexB/scancode-toolkit/issues/3465